test/e2e/tests/02-blank-form-fails.js

```diff
@@ -0,0 +1,88 @@
+const request = require('request');
+
+const report = require('../report');
+const config = require('../../../config');
+
+// FIXME: This should come in from a config variable
+const url = 'http://localhost:' + config.port + '/healthcare/apply/application/review-and-submit'; // eslint-disable-line
+
+// Test timeout constants
+// TODO(awong): Move to a common file usable for all tests.
+const timeouts = {
+  normal: 500,     // The normal timeout to use. For most opreations w/o a server roundtrip, this should be more than fast enough.
+  slow: 1000,      // A slow timeout incase the page is doing something complex.
+  molasses: 5000,  // A really really slow timeout. This should rarely be used.
+  submission: 10000 // Only to be used for submission.
+};
+
+// TODO(awong): Move this into a custom command or assertion that can be used with client.expect.element().
+function expectNavigateAwayFrom(client, urlSubstring) {
+  client.expect.element('.js-test-location').attribute('data-location')
+    .to.not.contain(urlSubstring).before(timeouts.normal);
+}
+
+const section = [
+  'confirm-personal-information',
+  'confirm-birth-information',
+  'confirm-demographic-information',
+  'confirm-veteran-address',
+  'confirm-contact-information',
+  'confirm-military-service-information',
+  'confirm-additional-service-information',
+  'confirm-basic-information',
+  'confirm-financial-disclosure',
+  'confirm-spouse-information',
+  'confirm-child-information',
+  'confirm-income-information',
+  'confirm-deductible-expenses',
+  'confirm-medicare-medicaid-information',
+  'confirm-insurance-information',
+  'confirm-additional-information'
+];
+
+module.exports = {
+  'Begin application': (client) => {
+    console.log(url);
+    request({
+      uri: `${url}/api/hca/v1/mock`,
+      method: 'POST',
+      json: {
+        resource: 'application',
+        verb: 'post',
+        value: {
+          formSubmissionId: '123fake-submission-id-567',
+          timeStamp: '2016-05-16'
+        }
+      }
+    });
+
+    // Review and Submit Page.
+    client
+      .url(url)
```

Comment:
This might be an indication that you're attempting to use an e2e test to do something a unittest is actually meant for.

What you seem to be trying to assert is "if redux store is blank and someone somehow manages to render the submit button and cilck it, we don't let it happen".  In that world, I'd want to just render the submit component using the Karma tests, manually set the store to something useless, and then test all those reactions.

Using an e2e test means your test "setup" becomes heavy weight for the assertion you are trying to make...